example/ios/Runner/Info.plist

```diff
@@ -45,5 +45,7 @@
 	<false/>
 	<key>CADisableMinimumFrameDurationOnPhone</key>
 	<true/>
+	<key>UIApplicationSupportsIndirectInputEvents</key>
+	<true/>
```

Comment:
what is it?

Reply:
From `iOS 13.4+` Info.plist automatically adds this. Please refer at [here](https://developer.apple.com/documentation/bundleresources/information_property_list/uiapplicationsupportsindirectinputevents)